schemas/test/en/test_progress_value_source_repeating_sections_chained_dependencies.json

```diff
@@ -0,0 +1,491 @@
+                            "type": "Question",
+                            "id": "second-other-question-block",
+                            "question": {
+                                "id": "second-other-question",
+                                "answers": [
+                                    {
+                                        "id": "second-other-answer",
+                                        "mandatory": true,
+                                        "label": "Anything",
+                                        "type": "Number"
+                                    }
+                                ],
+                                "title": {
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "person_name_possessive",
+                                            "transforms": [
+                                                {
+                                                    "arguments": {
+                                                        "delimiter": " ",
+                                                        "list_to_concatenate": [
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "second-first-name"
+                                                            },
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "second-last-name"
+                                                            }
+                                                        ]
+                                                    },
+                                                    "transform": "concatenate_list"
+                                                },
+                                                {
+                                                    "arguments": {
+                                                        "string_to_format": {
+                                                            "source": "previous_transform"
+                                                        }
+                                                    },
+                                                    "transform": "format_possessive"
+                                                }
+                                            ]
+                                        }
+                                    ],
+                                    "text": "Random question about <em>{person_name_possessive}</em>"
+                                },
+                                "description": ["Shows because the random question was completed in section 1"],
```

Comment:
I think this description should be updated to "Shows because section 2 has been completed"

Reply:
Updated 👍